### PR TITLE
Small grammar fixes and clarifications

### PR DESCRIPTION
**Pull request by joeybab3**
Just fixed some small grammar issues in the comments and clarified a few things that might seem confusing.

### CODE REVIEW

File: libraries/Preferences/examples/StartCounter/StartCounter.ino [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*
- ESP32 start counter example with Preferences library
+ ESP32 startup counter example with Preferences library.
 
- This simple example demonstrate using Preferences library to store how many times
- was ESP32 module started. Preferences library is wrapper around Non-volatile
+ This simple example demonstrates using the Preferences library to store how many times
+ the ESP32 module has booted. the Preferences library is a wrapper around the Non-volatile
```
Comment (stickbreaker) [inline libraries/Preferences/examples/StartCounter/StartCounter.ino]: Capitalization "the Preferences"

File: libraries/Preferences/examples/StartCounter/StartCounter.ino [hunk 2]
```diff
@@ -17,29 +17,29 @@ void setup() {
...
-  // Remove all preferences under opened namespace
+  // Remove all preferences under the opened namespace
   //preferences.clear();
 
   // Or remove the counter key only
   //preferences.remove("counter");
 
-  // Get a counter value, if key is not exist return default value 0
-  // Note: Key name is limited to 15 chars too
+  // Get the counter value, if the key doess not exist, return a default value of 0
```
Comment (stickbreaker) [inline libraries/Preferences/examples/StartCounter/StartCounter.ino]: SP "doess"

**Comment by joeybab3** [comment]
Thanks!

**Comment by me-no-dev** [comment]
merged :)

**Pull request merged**